scipy/optimize/_highs/setup.py

```diff
@@ -159,8 +160,8 @@ def configuration(parent_package='', top_path=None):
 
     # Export constants and enums from HiGHS:
     ext = config.add_extension(
-        'constants',
-        sources=['cython/src/constants.cxx'],
+        '_highs_constants',
```

Comment:
Good. Wanted to make sure this wasn't `_constants`, as I imagine that could be common, too.

Reply:
I don't think it should matter anymore, since with the newer changes, this is just a local name in the `_highs` module, it would be fine to even revert the name change entirely.  But it also doesn't hurt :).